app/views/articles/delete_confirm.html.erb

```diff
@@ -11,8 +9,7 @@
     </a>
     instead?
   </h2>
-
-  <h2>
-    <a class="delete-link" data-no-instant rel="nofollow" data-method="delete" href="/articles/<%= @article.id %>">DELETE</a>
-  </h2>
+  <%= form_tag "/articles/#{@article.id}", method: :delete do %>
```

Comment:
I don't remember, are these converted magically by Rails if we don't use ujs? Or browsers support `method=DELETE` ?